src/scss/06_components/_tags.scss

```diff
@@ -0,0 +1,53 @@
+$tag-gap: $spacing-half;
+
+.content-tags__tags {
+  @include flex-gap($tag-gap);
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: 0;
+  margin-top: 0;
+  padding-bottom: ($spacing-and-half - $tag-gap);
+  padding-left: 0;
+  padding-top: $spacing-and-half;
+}
+
+.content-tags__tags__tag {
+  list-style: none;
+  margin-bottom: $tag-gap;
+  margin-top: 0;
+
+  a,
+  span {
+    @include font('small');
+    display: inline-block;
+    line-height: 12px;
+    text-decoration: none;
+    transition: 0.2s ease-in-out all;
+  }
+}
+
+.content-tags__tags--interactive a {
+  background-color: transparent;
+  border: 2px solid $color-black;
+  border-radius: $spacing-and-half;
+  padding: $spacing 20px;
+
+  &:hover {
+    background-color: $color-black;
+    border-color: $color-black;
+    color: $color-white;
+  }
+
+  &:focus {
+    border-color: $color-black;
+    outline: 2px solid $color-black;
+    outline-offset: 2px;
+  }
```

Comment:
We should probably have active style too. Which is probably best to have as a combination of focus and hover styles. So just add the selector to both. This way safari gets the focus ring on activation like the rest of the browsers do too.